docs/Selectors/Link selector.md

```diff
@@ -6,22 +6,23 @@ the href attribute of the link. If you add child selectors to *Link selector*
 then these child selectors will be used in the page that this link was leading
 to. If you are selecting multiple links then check *multiple* property.
 
-**Note!** Link selector works only with `<a>` tags with `href` attribute. If the
-link selector is not working for you then you can try these workarounds:
+**Note!** Check that the link in the url bar changes after clicking an item. If the link doesn't change, then the site is
```

Comment:
Suggested changes:

```
Note! Check that the link in the URL bar changes after clicking on an item. If the link does not, then, most likely, the site is
using AJAX for data loading. Instead of using a link selector, you
should use [Element click selector] [element-click].
```